cpp.md

```diff
@@ -382,6 +382,15 @@ or we can for 3d vector like
 ```c
 vector<vector<vector<int>>> v3;
 ```
+Here are some useful vector functions you might need with short descriptions of each of them:  
+size() - Returns the size of the vector (1-based indexing)  
```

Comment:
Please Markdown table

Reply:
I changed my text to table format, please do check it out.